2019-Jun-09 18:02:35 log.py[line:146]/INFO/  Scrapy 1.6.0 started (bot: urlScrapy)
2019-Jun-09 18:02:35 log.py[line:149]/INFO/  Versions: lxml 4.3.3.0, libxml2 2.9.8, cssselect 1.0.3, parsel 1.5.1, w3lib 1.20.0, Twisted 19.2.0, Python 3.7.3 | packaged by conda-forge | (default, Mar 27 2019, 23:18:50) [MSC v.1900 64 bit (AMD64)], pyOpenSSL 19.0.0 (OpenSSL 1.1.1c  28 May 2019), cryptography 2.6.1, Platform Windows-10-10.0.17134-SP0
2019-Jun-09 18:02:35 crawler.py[line:38]/INFO/  Overridden settings: {'BOT_NAME': 'urlScrapy', 'CONCURRENT_REQUESTS': 10, 'CONCURRENT_REQUESTS_PER_DOMAIN': 10, 'CONCURRENT_REQUESTS_PER_IP': 10, 'DOWNLOAD_DELAY': 5, 'NEWSPIDER_MODULE': 'urlScrapy.spiders', 'ROBOTSTXT_OBEY': True, 'SPIDER_MODULES': ['urlScrapy.spiders']}
2019-Jun-09 18:02:35 telnet.py[line:60]/INFO/  Telnet Password: 5ab4fef34586d79d
2019-Jun-09 18:02:35 middleware.py[line:48]/INFO/  Enabled extensions:
['scrapy.extensions.corestats.CoreStats',
 'scrapy.extensions.telnet.TelnetConsole',
 'scrapy.extensions.logstats.LogStats']
2019-Jun-09 18:02:36 middleware.py[line:48]/INFO/  Enabled downloader middlewares:
['proxyPool.scrapy.middlewares.RetryMiddleware',
 'scrapy.downloadermiddlewares.robotstxt.RobotsTxtMiddleware',
 'proxyPool.scrapy.middlewares.ProxyMiddleware',
 'proxyPool.scrapy.middlewares.CatchExceptionMiddleware',
 'scrapy.downloadermiddlewares.httpauth.HttpAuthMiddleware',
 'scrapy.downloadermiddlewares.downloadtimeout.DownloadTimeoutMiddleware',
 'scrapy.downloadermiddlewares.defaultheaders.DefaultHeadersMiddleware',
 'proxyPool.scrapy.RandomUserAgentMiddleware.RandomUserAgentMiddleware',
 'scrapy.downloadermiddlewares.useragent.UserAgentMiddleware',
 'scrapy.downloadermiddlewares.retry.RetryMiddleware',
 'scrapy.downloadermiddlewares.redirect.MetaRefreshMiddleware',
 'scrapy.downloadermiddlewares.httpcompression.HttpCompressionMiddleware',
 'scrapy.downloadermiddlewares.redirect.RedirectMiddleware',
 'scrapy.downloadermiddlewares.cookies.CookiesMiddleware',
 'scrapy.downloadermiddlewares.httpproxy.HttpProxyMiddleware',
 'scrapy.downloadermiddlewares.stats.DownloaderStats']
2019-Jun-09 18:02:36 middleware.py[line:48]/INFO/  Enabled spider middlewares:
['scrapy.spidermiddlewares.httperror.HttpErrorMiddleware',
 'scrapy.spidermiddlewares.offsite.OffsiteMiddleware',
 'scrapy.spidermiddlewares.referer.RefererMiddleware',
 'scrapy.spidermiddlewares.urllength.UrlLengthMiddleware',
 'scrapy.spidermiddlewares.depth.DepthMiddleware']
2019-Jun-09 18:02:36 middleware.py[line:48]/INFO/  Enabled item pipelines:
['urlScrapy.pipelines.UrlscrapyPipeline']
2019-Jun-09 18:02:36 engine.py[line:256]/INFO/  Spider opened
2019-Jun-09 18:02:36 logstats.py[line:48]/INFO/  Crawled 0 pages (at 0 pages/min), scraped 0 items (at 0 items/min)
2019-Jun-09 18:02:36 telnet.py[line:74]/INFO/  Telnet console listening on 127.0.0.1:6023
2019-Jun-09 18:02:36 middlewares.py[line:19]/DEBUG/  =====  ProxyMiddleware get a random_proxy:【 http://221.4.150.7:8181 】 =====
2019-Jun-09 18:02:37 redirect.py[line:41]/DEBUG/  Redirecting (302) to <GET http://www.alexa.cn/plugin/tips?k=EgqZHDWUZVsBjZDc-JQD4FIsE6QIiDoCNfp9hEzpS4-AfcTrST6KXDkC8bM6bBU-AJm-J10D-JQv-AL-JDJaM2eZlnH6qgkA-AJTJmrkfAO-JhBQkqcnE7D-AkQS4Qr7xi7donNFzjPF3hfJbU0p5KxluXq0I3xyBTzQHpJ8fdNWNDETdVEUeSqhVwIg3hW5b-AvljxFdXOS5QhXXHjUFaevKZZfKiHcA-K-K> from <GET http://www.alexa.cn/robots.txt>
2019-Jun-09 18:02:37 middlewares.py[line:19]/DEBUG/  =====  ProxyMiddleware get a random_proxy:【 http://123.8.124.199:9999 】 =====
2019-Jun-09 18:03:04 retry.py[line:75]/DEBUG/  Retrying <GET http://www.alexa.cn/plugin/tips?k=EgqZHDWUZVsBjZDc-JQD4FIsE6QIiDoCNfp9hEzpS4-AfcTrST6KXDkC8bM6bBU-AJm-J10D-JQv-AL-JDJaM2eZlnH6qgkA-AJTJmrkfAO-JhBQkqcnE7D-AkQS4Qr7xi7donNFzjPF3hfJbU0p5KxluXq0I3xyBTzQHpJ8fdNWNDETdVEUeSqhVwIg3hW5b-AvljxFdXOS5QhXXHjUFaevKZZfKiHcA-K-K> (failed 1 times): TCP connection timed out: 10060: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。.
2019-Jun-09 18:03:04 middlewares.py[line:19]/DEBUG/  =====  ProxyMiddleware get a random_proxy:【 http://117.90.3.74:9000 】 =====
2019-Jun-09 18:03:25 retry.py[line:75]/DEBUG/  Retrying <GET http://www.alexa.cn/plugin/tips?k=EgqZHDWUZVsBjZDc-JQD4FIsE6QIiDoCNfp9hEzpS4-AfcTrST6KXDkC8bM6bBU-AJm-J10D-JQv-AL-JDJaM2eZlnH6qgkA-AJTJmrkfAO-JhBQkqcnE7D-AkQS4Qr7xi7donNFzjPF3hfJbU0p5KxluXq0I3xyBTzQHpJ8fdNWNDETdVEUeSqhVwIg3hW5b-AvljxFdXOS5QhXXHjUFaevKZZfKiHcA-K-K> (failed 2 times): TCP connection timed out: 10060: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。.
2019-Jun-09 18:03:25 middlewares.py[line:19]/DEBUG/  =====  ProxyMiddleware get a random_proxy:【 http://111.177.107.243:9999 】 =====
2019-Jun-09 18:03:36 logstats.py[line:48]/INFO/  Crawled 0 pages (at 0 pages/min), scraped 0 items (at 0 items/min)
2019-Jun-09 18:03:46 retry.py[line:91]/DEBUG/  Gave up retrying <GET http://www.alexa.cn/plugin/tips?k=EgqZHDWUZVsBjZDc-JQD4FIsE6QIiDoCNfp9hEzpS4-AfcTrST6KXDkC8bM6bBU-AJm-J10D-JQv-AL-JDJaM2eZlnH6qgkA-AJTJmrkfAO-JhBQkqcnE7D-AkQS4Qr7xi7donNFzjPF3hfJbU0p5KxluXq0I3xyBTzQHpJ8fdNWNDETdVEUeSqhVwIg3hW5b-AvljxFdXOS5QhXXHjUFaevKZZfKiHcA-K-K> (failed 3 times): TCP connection timed out: 10060: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。.
2019-Jun-09 18:03:46 proxyDBManager.py[line:181]/DEBUG/  ===  success to update 111.177.107.243 proxy  ===
2019-Jun-09 18:03:46 proxyDBManager.py[line:181]/DEBUG/  ===  success to update 111.177.107.243 proxy  ===
2019-Jun-09 18:03:46 robotstxt.py[line:85]/ERROR/  Error downloading <GET http://www.alexa.cn/robots.txt>: TCP connection timed out: 10060: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。.
Traceback (most recent call last):
  File "E:\Users\JohnParker\Anaconda3\lib\site-packages\scrapy\core\downloader\middleware.py", line 43, in process_request
    defer.returnValue((yield download_func(request=request,spider=spider)))
twisted.internet.error.TCPTimedOutError: TCP connection timed out: 10060: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。.
2019-Jun-09 18:03:46 middlewares.py[line:19]/DEBUG/  =====  ProxyMiddleware get a random_proxy:【 http://58.240.143.105:8118 】 =====
2019-Jun-09 18:03:52 retry.py[line:75]/DEBUG/  Retrying <GET http://www.alexa.cn/siterank/> (failed 1 times): Connection was refused by other side: 10061: 由于目标计算机积极拒绝，无法连接。.
2019-Jun-09 18:03:52 middlewares.py[line:19]/DEBUG/  =====  ProxyMiddleware get a random_proxy:【 http://60.13.42.208:9999 】 =====
2019-Jun-09 18:04:13 retry.py[line:75]/DEBUG/  Retrying <GET http://www.alexa.cn/siterank/> (failed 2 times): TCP connection timed out: 10060: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。.
2019-Jun-09 18:04:13 middlewares.py[line:19]/DEBUG/  =====  ProxyMiddleware get a random_proxy:【 http://111.177.107.73:9999 】 =====
2019-Jun-09 18:04:34 retry.py[line:91]/DEBUG/  Gave up retrying <GET http://www.alexa.cn/siterank/> (failed 3 times): TCP connection timed out: 10060: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。.
2019-Jun-09 18:04:34 proxyDBManager.py[line:181]/DEBUG/  ===  success to update 111.177.107.73 proxy  ===
2019-Jun-09 18:04:34 proxyDBManager.py[line:181]/DEBUG/  ===  success to update 111.177.107.73 proxy  ===
2019-Jun-09 18:04:35 scraper.py[line:208]/ERROR/  Error downloading <GET http://www.alexa.cn/siterank/>
Traceback (most recent call last):
  File "E:\Users\JohnParker\Anaconda3\lib\site-packages\scrapy\core\downloader\middleware.py", line 43, in process_request
    defer.returnValue((yield download_func(request=request,spider=spider)))
twisted.internet.error.TCPTimedOutError: TCP connection timed out: 10060: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。.
2019-Jun-09 18:04:35 engine.py[line:295]/INFO/  Closing spider (finished)
2019-Jun-09 18:04:35 statscollectors.py[line:47]/INFO/  Dumping Scrapy stats:
{'downloader/exception_count': 6,
 'downloader/exception_type_count/twisted.internet.error.ConnectionRefusedError': 1,
 'downloader/exception_type_count/twisted.internet.error.TCPTimedOutError': 5,
 'downloader/request_bytes': 3043,
 'downloader/request_count': 7,
 'downloader/request_method_count/GET': 7,
 'downloader/response_bytes': 370,
 'downloader/response_count': 1,
 'downloader/response_status_count/302': 1,
 'finish_reason': 'finished',
 'finish_time': datetime.datetime(2019, 6, 9, 10, 4, 35, 181167),
 'log_count/DEBUG': 18,
 'log_count/ERROR': 2,
 'log_count/INFO': 10,
 'retry/count': 4,
 'retry/max_reached': 2,
 'retry/reason_count/twisted.internet.error.ConnectionRefusedError': 1,
 'retry/reason_count/twisted.internet.error.TCPTimedOutError': 3,
 "robotstxt/exception_count/<class 'twisted.internet.error.TCPTimedOutError'>": 1,
 'robotstxt/request_count': 1,
 'scheduler/dequeued': 3,
 'scheduler/dequeued/memory': 3,
 'scheduler/enqueued': 3,
 'scheduler/enqueued/memory': 3,
 'start_time': datetime.datetime(2019, 6, 9, 10, 2, 36, 506979)}
2019-Jun-09 18:04:35 engine.py[line:326]/INFO/  Spider closed (finished)
